examples/source/1.components/amp-script/index.html

```diff
@@ -0,0 +1,138 @@
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <!--
+
+        ## Setup
+
+        First, you need to import the `amp-script` extension.
+      -->
+      <script async custom-element="amp-script" src="https://cdn.ampproject.org/v0/amp-script-0.1.js"></script>
+      <style amp-custom>
+        .amp-script-sample {
+          padding: 8px;
+        }
+        .amp-script-sample button {
+          background-color: #005AF0;
+          font-size: 16px;
+          font-weight: 700;
+          color: white;
+          border: none;
+          padding: 8px;
+          margin: 8px 0;
+        }
+      </style>
+
+      <!-- 
+        For inline scripts you need to generate a CSP header. Read more about this in the [component documentation](content/amp.dev/documentation/components/reference/amp-script.md)).
+      -->
+      <meta name="amp-script-src" content="sha384-X8xW7VFd-a-kgeKjsR4wgFSUlffP7x8zpVmqC6lm2DPadWUnwfdCBJ2KbwQn6ADE sha384-nNFaDRiLzgQEgiC5kP28pgiJVfNLVuw-nP3VBV-e2s3fOh0grENnhllLfygAuU_M sha384-u7NPnrcs7p4vsbGLhlYHsId_iDJbcOWxmBd9bhVuPoA_gM_he4vyK6GsuvFvr2ym">
+  </head>
+  <body>
+      <!-- 
+        ## Basic usage
+
+        An amp-script element can load a JavaScript file from a URL. This is usually the best way to integrate a script as the browser needs to download the script only once.
+      -->
+      <amp-script layout="container" src="<% hosts.platform %>/documentation/examples/components/amp-script/hello-world.js" class="amp-script-sample" >
+        <button id="hello">Click!</button>
+      </amp-script>
+
+      <!-- 
+        This is the script in [hello-world.js](hello-world.js):
+
+        ```js
+const button = document.querySelector('#hello');
+button.addEventListener('click', () => {
+  const h1 = document.createElement('h1');
+  h1.textContent = 'Hello World!';
+  document.body.appendChild(h1);
+});```
+
+        ## Inline Scripts
+
+        You can also declare scripts inline and reference them by `id`.
+      -->
+      <amp-script layout="container" script="hello-world" class="amp-script-sample">
+        <button id="hello2">Click!</button>
+      </amp-script>
+
+      <!-- 
+        This is the inlined script. Note that the type needs to be `type=text/plain` and the `target=amp-script`. 
+
+      -->
+      <script id="hello-world" type="text/plain" target="amp-script">
+    const button = document.querySelector('#hello2');
+    button.addEventListener('click', () => {
+      const h1 = document.createElement('h1');
+      h1.textContent = 'Hello World 2!';
+      document.body.appendChild(h1);
+    });
+  </script>
+
+  <!-- Note: Good to know: calling `document.body.appendChild(...)` will add an element to the `amp-script` element and not to the actual `body`. -->
```

Comment:
This makes it sound like `appendChild` is special, not `document.body`.

nit: I know `Note:` has special meaning and creates that callout box, but it's still weird to have two colons this close in a sentence. I'd remove "Good to know:".

Reply:
done